herwigpp-toolfile.spec

```diff
@@ -11,12 +11,16 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/herwigpp.xml
 <tool name="herwigpp" version="@TOOL_VERSION@">
   <client>
     <environment name="HERWIGPP_BASE" default="@TOOL_ROOT@"/>
-    <environment name="LIBDIR" default="$HERWIGPP_BASE/lib"/>
-    <environment name="INCLUDE" default="$HERWIGPP_BASE/include"/>
+    <environment name="LIBDIR" default="$HERWIGPP_BASE/lib/Herwig7"/>
+    <environment name="INCLUDE" default="$HERWIGPP_BASE/include/Herwig7"/>
   </client>
-  <runtime name="HERWIGPATH" value="$HERWIGPP_BASE/share/Herwig++"/>
+  <runtime name="HERWIGPATH" value="$HERWIGPP_BASE/share/Herwig7"/>
```

Comment:
@mharrend , please fix the toolfile to have correct path. There is no Herwig7 directory. I see only
```
slc6_amd64_gcc493/external/herwigpp/7.0.2/include/Herwig/ 
slc6_amd64_gcc493/external/herwigpp/7.0.2/lib/Herwig/  
slc6_amd64_gcc493/external/herwigpp/7.0.2/share/Herwig/
```